website/templates/project/retract_registration.mako

```diff
@@ -0,0 +1,46 @@
+<%inherit file="project/project_base.mako"/>
+<%def name="title()">Retract Registration of Component</%def>
+
+<legend class="text-center">Retract Registration</legend>
+
+    <div id="registrationRetraction" class="col-md-6 col-md-offset-3">
+        <div class="panel panel-default">
+            <div class="panel-body">
+                Retracting a registration will remove its content from the OSF, but leave basic meta-data behind.
```

Comment:
@bnosek @saradbowman Users will see this message while filling out the form to request the retraction of a registration